.circleci/config.yml

```diff
@@ -681,11 +681,18 @@ workflows:
 
       - python_job:
           name: python 3.12 test
-          python_version: "3.12"
+          python_version: "3.12.7"
           command: pytest
-          pytest_fail_fast: true
           test_results_filename: pytest-python-3-12.xml
+          filters: *default_filters
```

Comment:
question (non-blocking): Should we alphabetize these? Or at least make their order consistent in each python job?

Reply:
OK, I'm ordering the `python_job` job definition, then ordering the workflow definitions:

* `python_job.executor` parameters
  - `name`
  - `postgres_version` (as a matrix for the postgres jobs)
  - `python_version`
  - `variant`
* `python_job` parameters
  - `allow_fail`
  - `command`
  - `executor`
  - `mypy_strict`
  - `production_with_new_migrations`
  - `pytest_fail_fast`
  - `pytest_phones_backend`
  - `test_results_filename`
  - `update_dependency`
* `workflow` parameters
  - `filters`
  - (matrix could go here since it is a workflow parameter)